firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Homepage Rebuild/Redux/PocketMiddlewareTests.swift

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Redux
+import XCTest
+
+@testable import Client
+
+final class PocketMiddlewareTests: XCTestCase, StoreTestUtility {
+    let storeUtilityHelper = StoreTestUtilityHelper()
+    let pocketManager = MockPocketManager()
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies(injectedPocketManager: pocketManager)
```

Comment:
to test these middlewares (more of an integration test over unit test); all dependencies will need to be injected through the `DependencyHelper` and this is because of how we are initializing our middlewares.